### PR TITLE
Support SetVariable request

### PR DESCRIPTION
**Pull request by robertsipka**
Set the variable with the given name in the variable container to a new value

### CODE REVIEW

**Comment by robertsipka** [comment]
I've removed the WIP flag, ~~**PLEASE DO NOT MERGE**~~ until https://github.com/jerryscript-project/jerryscript/pull/2583 is not landed.

**Review by rtakacs — APPROVED** [review]
LGTM

**Comment by robertsipka** [comment]
The https://github.com/jerryscript-project/jerryscript/pull/2583 is landed, we could merge this PR after a second approve.

File: src/IotjsDebugger.ts
Comment (LaszloLango) [inline src/IotjsDebugger.ts]: This indentation looks odd to me. Is this the right style in this project?

File: src/test/JerryProtocolHandler.test.ts
Comment (LaszloLango) [inline src/test/JerryProtocolHandler.test.ts]: I think one new line is enough.

File: src/JerryProtocolHandler.ts
Comment (LaszloLango) [inline src/JerryProtocolHandler.ts]: I think one new line is enough.

**Comment by robertsipka** [comment]
@LaszloLango : Thanks for the review, I've updated the PR.

**Review by LaszloLango — APPROVED** [review]
LGTM

**Pull request merged**
